client/src/cordova/build.action.mjs

```diff
@@ -43,8 +45,6 @@ export async function main(...parameters) {
     );
   }
 
-  await runAction('client/src/www/build', ...parameters);
```

Comment:
The setup call already calls these actions. We were calling them twice!

Reply:
IIRC, this was intended? @daniellacosse